src/dsl/verifier/proxy/hooks.ts

```diff
@@ -31,19 +31,17 @@ export const registerAfterHook = (
   config: ProxyOptions,
   stateSetupPath: string
 ): void => {
+  logger.trace("registered 'afterEach' hook");
```

Comment:
This is now slightly misleading- it always registers the hook, but the hook is only connected to something if `config.afterEach` is set.